docs/manual/build_system/prerequisites.qbk

```diff
@@ -63,7 +63,7 @@ favorite compiler with __hpx__ visit __stellar_hpx_buildbot__.
     [[__cmake__                  ][3.9.0              ][3.3.2           ]]
 
     [[[*Required Libraries]]]
-    [[__boost_libraries__        ][1.57.0 or newer    ][1.55.0          ]]
+    [[__boost_libraries__        ][1.57.0 or newer    ][1.58.0          ]]
```

Comment:
Recommended version should presumably be at least as big as the minimum version?